src/java.xml/share/classes/javax/xml/xpath/XPath.java

```diff
@@ -98,6 +98,7 @@
  * method is invoked, applications may not recursively call
  * the {@code evaluate} method.
  *
+ * @spec https://www.w3.org/TR/xpath xpath cover page - W3C
  * @author  Norman Walsh
  * @author  Jeff Suttor
  * @see <a href="http://www.w3.org/TR/xpath">XML Path Language (XPath) Version 1.0</a>
```

Comment:
Similar situation as XSLT above, the latest version now points to "cover page". For this javadoc then, it needs to be:
    https://www.w3.org/TR/1999/REC-xpath-19991116/ XML Path Language (XPath) Version 1.0
    
Unlike XSLT, the original @see ref also points to the spec generally (not a specific section), we could then drop it to keep just the @spec ref.